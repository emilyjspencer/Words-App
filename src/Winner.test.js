import React from 'react';
import { shallow } from 'enzyme';

import Winner from './Winner';


describe('Winner', () => {

    let wrapper;

    beforeEach(() => wrapper = shallow(<Winner />));

    it('renders correctly', () => {
        expect(wrapper).toMatchSnapshot();
    });

    it('renders a <div />', () => {
        expect(wrapper.find('div').length).toEqual(1);
    });
});

