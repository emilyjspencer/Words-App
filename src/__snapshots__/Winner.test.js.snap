// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Winner renders correctly 1`] = `ShallowWrapper {}`;
